docs/usage/system_backend/namespace.rst

```diff
@@ -0,0 +1,79 @@
+Namespace
+=====
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Create Namespace
+----------------
+
+.. automethod:: hvac.api.system_backend.Namespace.create_namespace
+   :noindex:
+
+Examples
+````````
+
+.. testcode:: sys_namespace
+    :skipif: not test_utils.is_enterprise() 
+
+    import hvac
+    client = hvac.Client(url='https://127.0.0.1:8200')
+
+    # Create namespace team1 where team1 is a child of root
+    client.sys.create_namespace(path="team1")
+
+    # Create namespace team1/app1 where app1 is a child of team1
+    client2 = hvac.Client(url='https://127.0.0.1:8200', namespace="team1")
+    client2.sys.create_namespace(path="app1")
```

Comment:
For what it is worth, you can also have doctest validate the example output as well. E.g.:
```suggestion
    create_resp = client2.sys.create_namespace(path="app1")
    print('ID for created "app1" namespace: %s' % create_resp['data']['id'])

Example output:

.. testoutput:: sys_namespace
    
    ID for created "app1" namespace: ...
```

(Ellipses / `...` are treated as a wildcard)

The nice thing about including output in a `testoutput` directive is it helps ensure the examples are kept up to date with different Python and Vault versions if things change down the line. 

Not suggesting changing this set of documentation to use that directive is a requirement or anything, just saying :)

Reply:
yeah good point! the test is basically written so i think it makes sense to include the validation step! i was confused by it because, as a user, its just documentation so seeing JUST `testoutput` isnt helpful lol